# Use gzip kernel compression rather than xz. GZIP is almost as good as XZ for compression but is
# less CPU intense which CELES has little of.
--- a	2019-09-03 13:11:57.797329844 -0600
+++ b	2019-09-03 13:13:01.558943412 -0600
@@ -29,10 +29,10 @@
 CONFIG_HAVE_KERNEL_XZ=y
 CONFIG_HAVE_KERNEL_LZO=y
 CONFIG_HAVE_KERNEL_LZ4=y
-# CONFIG_KERNEL_GZIP is not set
+CONFIG_KERNEL_GZIP=y
 # CONFIG_KERNEL_BZIP2 is not set
 # CONFIG_KERNEL_LZMA is not set
-CONFIG_KERNEL_XZ=y
+# CONFIG_KERNEL_XZ is not set
 # CONFIG_KERNEL_LZO is not set
 # CONFIG_KERNEL_LZ4 is not set
 CONFIG_DEFAULT_HOSTNAME="archlinux"
